Dockerfile.server

```diff
@@ -1,11 +1,10 @@
-FROM python:3.8.12-bullseye
```

Comment:
whats the difference between bullseye and not?

Reply:
I *think* it has to do with the base OS, debian vs. Ubuntu etc. will check on that